src/Schema/Types/GraphQLSubscription.php

```diff
@@ -16,7 +16,7 @@ abstract class GraphQLSubscription
      * @param  \Nuwave\Lighthouse\Subscriptions\Subscriber  $subscriber
      * @return bool
      */
-    public function can(Subscriber $subscriber)
+    public function can(Subscriber $subscriber): bool
```

Comment:
Those are breaking changes, since the class is meant to be extended. Let's add a TODO and change that in the next major version.